tests/conftest.py

```diff
@@ -2,40 +2,9 @@
 import pytest
 import yaml
 
-from roman_datamodels import datamodels
-from roman_datamodels import maker_utils as utils
-
 MANIFEST = yaml.safe_load(asdf.get_config().resource_manager["asdf://stsci.edu/datamodels/roman/manifests/datamodels-1.0"])
 
 
 @pytest.fixture(scope="session")
 def manifest():
     return MANIFEST
-
-
-@pytest.fixture(name="set_up_list_of_l2_files")
```

Comment:
Are the two functions here unused? Looking back at the history they were added in #143 but I'm not seeing where they were used.

Reply:
They were totally ununsed.

Reply:
The back history is that this was used by, but never added by the `ModelContainer` which was removed by PR #204. Clearly #204 missed removing this. The #143 PR added it so that the `ModelContainer` tests would actually be run correctly (`pytest` was raising a clear warning about this).